ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/ontologies/views/dialogs/EdamUploadDialog.java

```diff
@@ -0,0 +1,219 @@
+package org.iplantc.de.admin.desktop.client.ontologies.views.dialogs;
```

Comment:
Is there way to reuse / refactor 'SimpleFileUploadDialog' class instead of duplicating the whole logic here ?